homeassistant/components/mqtt/strings.json

```diff
@@ -2,7 +2,7 @@
   "config": {
     "step": {
       "broker": {
-        "description": "Please enter the connection information of your MQTT broker.",
+        "description": "Please enter the connection information of your MQTT broker and select if MQTT discovery should be enabled (recommended). If discovery is enabled, Home Assistant will automatically discover devices and entities which publish their configuration on the MQTT broker. If discovery is disabled, all configuration must be done manually.",
```

Comment:
I would suggest we remove it from initial config flow and just make it only part of the options flow to disable.

Reply:
Oh, that I like. Just enable the option by default. Its not critical to the config flow to complete.

Reply:
Updated